tests/test_inheritance.rs

```diff
@@ -112,16 +112,17 @@ fn mutation_fails() {
 #[test]
 fn is_subclass_and_is_instance() {
     Python::with_gil(|py| {
-        let sub_ty = py.get_type::<SubClass>();
-        let base_ty = py.get_type::<BaseClass>();
+        let sub_ty = py.get_type_bound::<SubClass>();
+        let base_ty = py.get_type_bound::<BaseClass>();
         assert!(sub_ty.is_subclass_of::<BaseClass>().unwrap());
-        assert!(sub_ty.is_subclass(base_ty).unwrap());
+        assert!(sub_ty.is_subclass(&base_ty).unwrap());
 
+        // FIXME: Whats the new API to do this?
```

Comment:
I think you're looking for `Py::new` or `Bound::new`, I guess we should deprecate `PyCell::new`in favour of one of these.

Reply:
Ah, nice and easy! (And thanks for catching, I totally forget that i put that there)

> I guess we should deprecate `PyCell::new` in favour of one of these.

That's a good idea, I'll look into that 👍